tests/system/test_table_admin.py

```diff
@@ -323,6 +324,27 @@ def test_table_backup(
         temp_table_backup.expire_time
     )
 
+    # Testing `backup.copy()` method
+    backup_op = temp_backup.copy("copied-backup")
```

Comment:
I would also add some test cases with some of the other parameters set.

Reply:
I've added tests that pass additional parameters.